### ISSUE TITLE
No hackrf support

### ISSUE THREAD

**Issue by uniformsierra** [polygon/sdr-power#2]
Hi,
thank you for your work but the hackrf doesn't work: unsupported radio.
I note that the bladerf is also reported to not work.
If this is the case, what radios are support (other than rtl)?

I was super pumped to get rtl_power features working for my HackRF One!!

Thanks again,
UniS

**Comment by polygon** [comment]
The is now support for the hackrf. Can you try and see if it works for you?

**Comment by uniformsierra** [comment]
Hi polygon,
Thanks very much for your help!
I have tried and it looks to work but the output file is always empty...
I had a few errors during make, I'll try again and document properly.

**Comment by uniformsierra** [comment]
Hi polygon,
reinstalled (from bare-metal) and everything seems to work great!!
I'm really happy to start playing so thanks very much for your efforts.

note - if using on raspberry pi you need to force the use of cmake 3.2.3

wget http://www.cmake.org/files/v3.2/cmake-3.2.3.tar.gz
tar xzf cmake-3.2.3.tar.gz
cd cmake-3.2.3
./configure
make install

Appreciated.

**Comment by superkuh** [comment]
Nah, you can even use cmake 2.8. Just change the file sdr-power/CMakeLists.txt so that,

cmake_minimum_required(VERSION 3.2)
reads
cmake_minimum_required(VERSION 2.8)
